go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	k8s.io/code-generator v0.22.5
 	knative.dev/caching v0.0.0-20220302153644-5a6403ef2de5
 	knative.dev/eventing v0.30.0
-	knative.dev/hack v0.0.0-20220224013837-e1785985d364
-	knative.dev/pkg v0.0.0-20220301181942-2fdd5f232e77
+	knative.dev/hack v0.0.0-20220302134343-0687d6e8cd79
+	knative.dev/pkg v0.0.0-20220302134643-d2cdc682d974
```

Comment:
hash for pkg for the 1.3 release is [2fdd5f2](https://github.com/knative/pkg/commit/2fdd5f232e77efb2afd214ddc89e0b341508a700)

Reply:
This is automatically generated by running the script: ./hack/update-deps.sh --upgrade --release v0.30